soc/arm/st_stm32/common/Kconfig.soc

```diff
@@ -0,0 +1,12 @@
+# Kconfig - ST Microelectronics Common Kconfig
+#
+# Copyright (c) 2019 Linaro Ltd.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
```

Comment:
Could clean up header (similarly to https://github.com/zephyrproject-rtos/zephyr/pull/20280).

```
# ST Microelectronics Common Kconfig

# Copyright (c) 2019 Linaro Ltd.
# SPDX-License-Identifier: Apache-2.0
```

Reply:
Done